book/docs/Open_data_code.ipynb

```diff
@@ -2,21 +2,77 @@
  "cells": [
```

Comment:
<p>Same pystac client fix as the other notebook. See <a href="https://github.com/developmentseed/asdi-examples/blob/main/amazonia-1/amazonia-1-tiler-example.ipynb" target="_blank">another example</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>